pkg/webui/console/containers/device-onboarding-form/device-provisioning-form-section/index.js

```diff
@@ -120,6 +121,14 @@ const DeviceProvisioningFormSection = () => {
     setFieldValue('_claim', supportsClaiming)
   }, [ids, setFieldValue, dispatch])
 
+  useEffect(() => {
+    if (_withQRdata) {
+      if (ids.join_eui?.length === 16) {
+        handleJoinEuiConfirm()
+      }
+    }
```

Comment:
```suggestion
    if (_withQRdata) {
      handleJoinEuiConfirm()
    }
```
The QR code always contains the join EUI anyway.